--- server.js.orig	2016-07-30 20:24:58.594831016 +0300
+++ server.js	2016-08-05 22:09:15.016312855 +0300
@@ -115,7 +115,7 @@
 
     const argv = minimist(process.argv.slice(2));
     const port = parseInt(argv.port, 10) || parseInt(process.env.PORT, 10) || 3002;
-    app.listen(port);
+    app.listen(port, '127.0.0.1');
 
     // Tell parent process koa-server is started
     if (process.send) process.send('online');
